src/Tests/Runtime/LruCacheTests.cs

```diff
@@ -75,6 +75,8 @@ public void ClearsOnTimeout()
             }
             Thread.Sleep(30); // let's give the timer some margin so this test does not fail randomly
             GC.Collect(2, GCCollectionMode.Forced);
+            Thread.Sleep(30);
```

Comment:
we can try both :sweat_smile: 

I don't think it will make much of difference, since blocking is true by default: https://source.dot.net/#System.Private.CoreLib/src/System/GC.cs,184